.github/workflows/wix.yaml

```diff
@@ -34,6 +36,59 @@ jobs:
     - run: npm run build -- --win --publish=never
       env:
         RD_FEAT_WIX: '1'
+    - name: Upload Windows installer
+      uses: actions/upload-artifact@v3
+      with:
+        name: Rancher Desktop Setup unsigned.msi
+        path: dist/Rancher Desktop*.msi
+        if-no-files-found: error
+    - name: Upload Windows zip
+      uses: actions/upload-artifact@v3
+      with:
+        name: Rancher Desktop-win.zip
+        path: dist/Rancher Desktop-*-win.zip
+        if-no-files-found: error
+  sign:
+    name: Test Signing
+    needs: package
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        persist-credentials: false
+    - name: Install Windows dependencies
+      shell: powershell
+      run: .\scripts\windows-setup.ps1 -SkipVisualStudio -SkipTools
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
```

Comment:
We should probably point all our go versions to a single place, right now they are scattered everywhere.

Reply:
Yes, but this workflow will be removed (and merged with the main packaging flow) soon, so there's no need to think about that over here.